docs/appendices/release-notes/unreleased.rst

```diff
@@ -86,6 +86,12 @@ Changes
 
 - Added support of ``numeric`` type to the ``avg`` aggregation function.
 
+- Enabled HTTP connections to persist :ref:`session settings <conf-session>`
+  across the requests, so that the session settings will be persisted if the
+  same connection is re-used. One of the scenarios that the session settings
+  cannot be persisted includes a multi-node setup with the incoming requests
+  re-distributed in round-robin.
```

Comment:
```suggestion
- Enabled HTTP connections to preserve :ref:`session settings <conf-session>`
  across the requests as long as the connection is re-used.

  Note that this is done on a node to node basis. If a client sends requests to
  different nodes, those won't share the same session settings, unless the
  client sets the session settings on each node individually.
```

I'd avoid the word `persist` as it is usually used in the context of storing something permanently (e.g. to disk). Session settings are still ephemeral - but their lifetime is now bound to the connection instead of per request.

Reply:
much easier to read, Thx. :+1: